### PR TITLE
Checkpoint

### PR DESCRIPTION
**Pull request by MaggieAHewitt**

### CODE REVIEW

File: unity-project/Assets/Scripts/BoundaryCollision.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		
+	}
+	
+	// Update is called once per frame
+	void OnCollisionEnter(Collision collision)
+	{
+		if(collision.gameObject.tag == "Player"){
+			Debug.Log("Collision detected");
+			gameController.RespawnPlayer(collision.gameObject);
+			//Destroy(this);
```
Comment (bspates) [inline unity-project/Assets/Scripts/BoundaryCollision.cs]: Is this no longer needed?

File: unity-project/Assets/Scripts/GameController.cs [hunk 2]
```diff
@@ -17,11 +17,29 @@ public class GameController : MonoBehaviour {
...
 	void Start () {
 
+
 		GameObject collectablesObject = GameObject.FindWithTag ("Collectables");
+
+		GameObject[] objs = GameObject.FindGameObjectsWithTag ("Water");
+		all_collectables = new ArrayList ();
+		collected_collectables = new ArrayList();
+
+		Debug.Log ("Collectables on load: " + objs.Length);
```
Comment (bspates) [inline unity-project/Assets/Scripts/GameController.cs]: Needed?

**Comment by bspates** [comment]
LGTM

**Comment by gmb9280** [comment]
LGTM :)

**Comment by CptTime** [comment]
LGTM

**Pull request merged**